ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -266,7 +270,12 @@ fi
 #
   lbc_spec_fhrs=( "" )
 
-  if [ "${anl_or_fcst}" = "FCST" ]; then
+  if [ "${anl_or_fcst}" = "ANL" ]; then
+    ic_spec_fhrs=$(( 0 + time_offset_hrs ))
```

Comment:
Why is this adding instead of subtracting?

Reply:
Please put that in a comment.

Reply:
Done